roles/ec2-ami-base/files/salt/salt-minion.conf

```diff
@@ -0,0 +1,22 @@
+description "Salt Minion"
+
+start on (net-device-up
+          and local-filesystems
+          and runlevel [2345])
+stop on runlevel [!2345]
```

Comment:
Can we not remove all the commented out stuff from here and just keep the nessecary config, it's a tl;dr at the moment